lib/config/executor/network_interface.go

```diff
@@ -66,4 +67,6 @@ type ContainerNetwork struct {
 
 	// set of network wide links and aliases for this container on this network
 	Aliases []string `vic:"0.1" scope:"hidden" key:"aliases"`
+
+	PortGroup object.NetworkReference `vic:"0.1" scope:"read-only" recurse:"depth=0"`
```

Comment:
Didn't think this change will impact the iso file size. Then it's definitely not good.
In theory, the PortGroup is not part of configuration, it's just stay inside of configuration structure for internal cache only.
So I will change the network context structure to separate it.